book/02-using-atom/sections/06-customizing.asc

```diff
@@ -85,7 +85,7 @@ You can open this file in an editor from the _Atom > Open Your Config_ menu.
 ** `nonWordCharacters`: A string of non-word characters to define word boundaries
 ** `fontSize`: The editor font size
 ** `fontFamily`: The editor font family
-** `invisibles`: Specify characters that Atom renders for invisibles in this hash (use `false` to turn off individual character types)
+** `invisibles`: Specify characters that Atom renders as invisible in this hash (use `false` to turn off individual character types)
```

Comment:
The purpose of the `invisibles` object is to specify how whitespace characters (such as spaces, tabs, and newlines) will be rendered. For example:

```
  editor:
    invisibles:
      space: "O"
```

makes space characters show up as "O":

![screen shot 2015-07-22 at 16 56 16](https://cloud.githubusercontent.com/assets/38924/8828458/9a63455e-3092-11e5-8a58-692707d9da8c.png)

Does that match the meaning of the sentence after your changes? To me, it sounds like you're saying that these characters will be made invisible by the editor, which isn't the same thing (and is not true).